examples/azure/terraform-azure-appserviceplan-example/output.tf

```diff
@@ -0,0 +1,3 @@
+output "planids" {
+  value = "${azurerm_app_service_plan.plan.*.id}"
```

Comment:
Do you intend this to be a list? Or a string? If string, might be better to join them using some delimitter.